src/components/Login/Login.tsx

```diff
@@ -3,7 +3,7 @@ import { LoginMain } from "./LoginStyled"
 
 const Login = ( ): JSX.Element => {
   return (
-    <LoginMain>THIS IS A TEST</LoginMain>
+    <LoginMain>THIS IS A TEST this is awesome</LoginMain>
```

Comment:
What's the purpose for this commit change if it's really no different than the previous commit, other than some added text?

Reply:
Do not merge